lib/user.rb

```diff
@@ -0,0 +1,15 @@
+class User
+
+  attr_reader :name, :jokes, :learn
+  def initialize(name, jokes = [])
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke_1)
```

Comment:
Remember that your method is just a blueprint, so it is _ok_ to call this variable `joke_1`, but it would make more sense to call it `joke` since it will always be a joke object (regardless of the variable that is holding on to it in the test or interaction pattern)